base/applications/iexplore/iexplore.rc

```diff
@@ -27,7 +27,7 @@
 #include "wine/wine_common_ver.rc"
 
 /* @makedep: iexplore.ico */
-1 ICON iexplore.ico
+1 ICON "iexplore.ico"
```

Comment:
Does adding the quotations fix anything specific?

Reply:
Reverted.